front/app/components/CustomFieldsForm/CustomFields.tsx

```diff
@@ -96,6 +97,8 @@ const renderField = ({
       return <RatingField question={question} />;
     case 'matrix_linear_scale':
       return <MatrixField question={question} />;
+    case 'cosponsor_ids':
+      return <CosponsorsField question={question} />;
```

Comment:
We need to add existing co-sponsors as a default value here to make sure editing works as expected.